multistreaming-server/rtmp-conf-generator.py

```diff
@@ -73,135 +34,191 @@
 PUSH_URL_MICROSOFT_STREAM = "%%RTMP_URL%% app=live/%%APP_NAME%%"
 PUSH_URL_MIXCLOUD = "rtmp://rtmp.mixcloud.com/broadcast/%%STREAM_KEY%%"
 PUSH_URL_DLIVE = "rtmp://stream.dlive.tv/live/%%STREAM_KEY%%"
-#
-#
-#
+
+
+DEFAULT_TRANSCODE_CONFIG = {
+    "pixels": "1280x720",
+    "videoBitRate": "4500k",
+    "videoFrameRate": 60,
+    "keyFrames": 60,
+    "audioOpts": "-c:a copy",
+    "logFfmpeg": True if CONFIG_FFMPEG_LOG else False,
+    "applicationEndpoints": set(),
+    "maxMuxingQueueSize": int(CONFIG_FFMPEG_MAX_MUXING_QUEUE_SIZE)
+    if CONFIG_FFMPEG_MAX_MUXING_QUEUE_SIZE
+    else None,
+}
+
+DEFAULT_AUDIO_OPTS = {
+    "audioBitRate": "160k",
+    "audioSampleRate": "48000",
+}
+
 
 def generatePlatormPushURL(block_config):
-    if 'platform' not in block_config:
+    if "platform" not in block_config:
         print("ERROR - Application block is missing platform element.", file=sys.stderr)
         exit(1)
-    push_url = 'push-it-real-good'
-    if block_config['platform'] == 'youtube':
-        push_url = PUSH_URL_YOUTUBE.replace('%%STREAM_KEY%%', block_config['streamKey'])
-    elif block_config['platform'] == 'facebook':
+    push_url = "push-it-real-good"
```

Comment:
Not trying to have a 'style war' war here (really), but what is the thinking behind changing the `'` to `"`? All of the changes just to change the string indicator makes this hard to review.

Reply:
Oh, really no reason (i swear!). I ran the entire file through [black ](https://github.com/psf/black) to just straighten out style inconsistencies. I should be able to switch back to single quotes easily.

Reply:
Switched everything back to single quotes.